.github/workflows/publish.yml

```
@@ -15,15 +15,15 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v4
         with:
-          node-version: 14
-          cache: 'yarn'
-          registry-url: 'https://registry.npmjs.org'
+          node-version: 20
+          cache: 'npm'
+          registry-url: 'https://npm.pkg.github.com'
```

I doesn't look like this is supposed to change.
```suggestion
          registry-url: 'https://registry.npmjs.org'
```
It should publish to the npm registry and github packages, right?

---

Good catch. Thanks